src/gapp_login/sms/config.py

```diff
@@ -6,10 +8,33 @@
 
 config: Config = load_entry_point("config", Config)
 
+
+def get_sms_provider(settings: str):
+    """Configs used to create a SMS provider instance. Sample:
+    {
+        "params": {
+            "secret_id": "xxx",
+            "secret_key": "xxx",
+            "sms_app_id": "234123",
+            "sms_template_id": "123456"
+            "sms_sign": "HiDay"
```

Comment:
I'd suggest making these config values separate ones in a separate config file for "Tencent SMS" only.

Reply:
fixed